content/en-us/general/mining/paas-miner.md

```diff
@@ -13,9 +13,9 @@ menu:
 
 ## Before Getting Started
 
-1. Sign up on <a href="https://www.vultr.com/products/bare-metal/"><img alt="VULTR Bare Metal" src="/images/docs/quick-start/mine-phala/signet__on-dark-blue-bg.png" width="30"> [Vultr](https://vultr.com).
+1. Sign up on <a href="https://www.vultr.com/?ref=9108833-8H"><img alt="VULTR Bare Metal" src="/images/docs/quick-start/mine-phala/signet__on-dark-blue-bg.png" width="30"> [Vultr](https://vultr.com).
```

Comment:
What's the `ref` for?

Reply:
Good catch! Vultr currently gives our 100 USD credit for new signups. This might be a good incentive for new miners.